.github/workflows/publish-to-pypi.yml

```diff
@@ -28,6 +28,8 @@ jobs:
       run: pylint3 wheatley/**/*.py
     - name: Run tests
       run: python -m pytest
+    - name: Run integration tests
```

Comment:
Possibly not worth running these here. 
As they rely on making a connection to ringing room and the method library, there probably will be a failures due to this at some point.
They run both before a pull request is merged and once pushed already.

Reply:
Yeah OK that sounds like a good plan.